### DISCUSSION TITLE
Use SML / OBIS while using 8E1 on UART / serial

### DISCUSSION THREAD

**Discussion by hagenbuch** [arendst/Tasmota#21037] [Q&A] [answered]
I have read that there is a straightforward way to switch between 8N1 and 8E1 (meaning "8 bits, No or Even parity, 1 stop bit") when using Modbus, great!

However, I would need this feature to access an Itron SML / OBIS energy meter (5 II 1 alias ACE6000 DC4). The energy company told me that this counter uses 8E1 9600 Baud.

Someone decided to recompile, hoewever it would be useful to be able to configure the serial parameter(s) in the script. Did I overlook something? Could maybe the "baud parameter" be upgraded to something like "9600@8E1" ?

**Comment by Wintespe** [comment]
Refer to
https://tasmota.github.io/docs/Commands/#serial-bridge
**SerialConfig 8E2
Baudrate 9600**

**Reply by hagenbuch** [reply]
Many thanks! I hope this works with the SML scripting, too. Will try and report here, thanks again!

**Reply by gemu2015** [reply, marked as answer]
this will not work at all,  serial bridge is unrelated to sml.

just append mode to sml type
oE1 instead of o

**Reply by hagenbuch** [reply]
Even better, many thanks, will try!
